### ISSUE TITLE
Unable to signup/login from client (appimage)

### ISSUE THREAD

**Issue by g-kartik** [gitbutlerapp/gitbutler#2975]
When I click on the signup/ login button, nothing happens. I have both firefox and chrome installed on my system. I am using Arch linux and running 0.10.19 version of GitButler.

**Comment by mtsgrd** [comment]
When you've got this in your build then you should at least get a toast showing the URL and error message: #2989

**Comment by Byron** [comment]
This issue might be related to #2656 .

Is this still the case with the latest Version 0.11.3 (20240419.070253)? And if it is, do see anything suspicious [in the log files](https://docs.gitbutler.com/development/debugging#logs)? Thanks for your help getting to the bottom of this.

**Comment by talal** [comment]
@Byron 

This issue still exists in the latest version (0.11.5). 

With a fresh install of the v0.11.5 AppImage on Fedora. The   `~/.config/com.gitbutler.app/logs/ui-logs.log` has a repetition of the following entry, I'm guessing one for each time I tried to open an external link like login:

```
[2024-05-02][13:11:37][ERROR][webview] Unhandled exception: shell error: failed to open: Scoped shell IO error: exit status: 3 undefined
```

**Comment by qhrille242** [comment]
@Byron This is still an issue in the latest appimage 0.12.5

Running on Fedora 40.

`[2024-06-25][07:36:59][ERROR][webview] Unhandled exception: shell error: failed to open: Scoped shell IO error: exit status: 3 undefined
`

**Comment by Byron** [comment]
Thanks a lot, I think it's clear now that it's a problem with the AppImage specifically.

**Comment by Byron** [comment]
Let's track this in #4149 which is a newer version of the same issue, with all we know conveniently gathered.

**Comment by krlvi** [comment]
Based on this conversation, it may be the case that this issue is not yet fixed: https://discord.com/channels/1060193121130000425/1275922675809783891/1275928310790099005

I will reopen this so that we can keep an eye until we are sure sure

**Comment by ndom91** [comment]
I would love to know if folks are still running into this issue with the latest releases (https://app.gitbutler.com/download).

I'm able to reproduce the broken xdg-open behavior on my local machine with NixOS. However, the AppImage open external link functionality works without issue on an Ubuntu VM of mine. 

If you could mention your distro and version that would help me narrow down the platforms a lot :pray:

**Comment by ndom91** [comment]
Just wanted to mention here that it seems like this is a tauri-wide issue where the `xdg-open` bundled in from the build environment (Ubuntu 22.04 in our case from the GHA runner) runs into errors when executed on other distros. 

See: https://github.com/tauri-apps/tauri/issues/10078

We're working on a workaround at GitButler now :pray:

**Comment by ndom91** [comment]
If anyone is running into this still, I'd love some feedback on a recent build. It works for me on a bunch of distros I tested including:

- My local NixOS machine
- Manjaro VM (fresh boot off their latest live CD)
- Ubuntu 22.04 VM (fresh boot off their latest live CD)
- Fedora 39 VM (fresh boot off their latest live CD)

git-butler-nightly_0.0.0_amd64.AppImage - [we.tl/t-Y2xR6TSdPl](https://we.tl/t-Y2xR6TSdPl)

**Comment by ndom91** [comment]
This has been fixed and merged to main last night. It's available in nightly `0.5.731+` (https://app.gitbutler.com/downloads).

Let us know if you have any issues with it :pray:

**Comment by ndom91** [comment]
I'm going to close this issue as I haven't heard back from anyone that it doesn't work for them. Of course if yuo're still having issues, please feel free to open a new issue. Make sure to include which Distro, desktop environment / window manager, etc. that yuo're using :pray:
